### PR TITLE
Implemented Sign In With Solana

### PR DESCRIPTION
**Pull request by pawartur**
, using the Phantom Wallet

### CODE REVIEW

**Review by oed — COMMENTED** [review]
Why is this not using the [`@didtools/pkh-solana`](https://did.js.org/docs/api/modules/pkh_solana)??

**Comment by oed** [comment]
Same thing for Ethereum actually :)
https://did.js.org/docs/api/modules/pkh_ethereum

**Comment by pawartur** [comment]
Thank You, @oed ! I hadn't noticed I was using an old version of did-session.

File: src/components/DIDSessionDemoHandler.tsx [hunk 1]
```diff
@@ -19,24 +20,44 @@ const MessageTextField = styled(TextField)({
...
-    const oneWeek = 60 * 60 * 24 * 7
-
-    const session = await DIDSession.authorize(
-      authProvider,
-      {
-        resources: ['test-resource', 'another-test-resource'],
-        expiresInSecs: oneWeek,
-        domain: 'YourAppName'
-      })
+    const addresses = await (ethProvider as any).enable()
```
Comment (oed) [inline src/components/DIDSessionDemoHandler.tsx]: I think this is the proper way to do it still!
```suggestion
    const addresses = await (ethProvider as any).request({ method: 'eth_requestAccounts' })
```

Comment (oed) [reply src/components/DIDSessionDemoHandler.tsx]: We should probably update the docs.

**Review by oed — APPROVED** [review]
lgtm, thanks for adding and fixing!

One minor comment

File: src/components/DIDSessionDemoHandler.tsx (hunk 1 above)
Comment (pawartur) [reply src/components/DIDSessionDemoHandler.tsx]: Will do! Thanks, Joel!

**Pull request merged**